components/docs/ruleOpsTable.tsx

```diff
@@ -64,14 +73,27 @@ const DataTable = ({ data, heading, eventKey }) => {
       <Accordion.Collapse eventKey={eventKey}>
         <Fragment>
           <p>
+            {heading === "date" ? (
+              <Alert variant="primary">
+                💡 Dates used for calculated groups should use full ISO-8601
+                format: <br />
+                <span className="pl-4">
+                  <b>YYYY-MM-DDThh:mm:ss.mmm+TZD</b>{" "}
+                  ("2020-09-01T08:15:00-8:00")
```

Comment:
I'd suggest using `+00:00` in your examples, keeping everything in UTC. Most databases are in UTC

Reply:
But I think the warning you added about dates and your example format is 👍

Reply:
> I think this is great!
> 
> I agree that everything probably needs an example. There's another story (https://www.pivotaltracker.com/story/show/177599093) for that, but please feel free to pick it up now as well! Examples should be added to the table (like you are doing now), and also higher up on the page we have an example (`async function buildConfig`) with no rules. Please add a group that shows off some of the more interesting rules, like "New High Value customers) - has an email address, isn't from @grouparoo.com, has signed up within 7 days, etc.

Would you like me to add those additional examples and such to this PR or a new one?  I've been working through them Friday/this morning.

Reply:
Totally up to you - whichever is easier at this point.  It's realated enough work that being in the same PR would be OK